### PR TITLE
LRDOCS-7665 - specify which product option field types 

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/customizing-your-product-with-product-options.md [hunk 1]
```diff
@@ -32,8 +32,16 @@ When _Use in Faceted Navigation_ is switched to _Active_, users have the ability
...
+* Select From List
+* Single Selection
+* Multiple Selection
+
+Otherwise if selecting one of the others, the following warning message appears:
+
 ```warning::
-   You cannot use a Boolean option as a SKU contributor. Attempting to use a Boolean option as a SKU contributor shows the following warning message: ``SKU Contributor cannot be set as true for the selected Field Type.``
+    ``SKU Contributor cannot be set as true for the selected Field Type.``
 ```
```
Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/customizing-your-product-with-product-options.md:45]: Error messages is not an appropriate use of an Admonition. Please refactor the error message out of the admonition.

Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/customizing-your-product-with-product-options.md:45]: After reviewing the article as a whole, I think we should consider adding a new H2 (or H3) section to present information about using Option Fields as SKU Contributors. Right now it's all tagged on as added information at the bottom of "Creating an Option Template" and this is starting to feel excessive.

**Comment by jrhoun** [comment]
Please review/re-send. Let me know if you have questions about my remarks. Thank you!

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/customizing-your-product-with-product-options.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/customizing-your-product-with-product-options.md:45]: Revised

Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/products/customizing-your-product-with-product-options.md:45]: Added h3 heading.

**Pull request closed without merging**